openshift-container-platform-4/policies/AC-Access_Control/component.yaml

```diff
@@ -232,16 +232,23 @@
 - control_key: AC-2 (11)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: partial
+  implementation_status: complete
   narrative:
     - text: |-
-        This control will only be partially satisfied when OpenShift is
-        configured to connect to an Identity Provider (IdP).
-
-        More work is being done to address this control. Engineering
-        progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-210
+       This functionality would fall to third
+       party software such as an Identity Provider (IdP),
+       Active Directory (MSAD), LDAP, etc.
+
+       To configure an IdP, see https://docs.openshift.com/container-platform/latest/authentication/identity_providers/configuring-ldap-identity-provider.html#configuring-ldap-identity-provider
+
+       For active alerting on auditable events then then
+       OpneShift audit log system (https://docs.openshift.com/container-platform/latest/nodes/nodes/nodes-nodes-audit-log.html)
+       would need to be enabled
+       and the OpenShift Elasticsearch, Kibarana,
+       and FluentD (EFK) stack configured to alert on the
+       organizational specified audit events, or a third party
+       tool configured to consume the OpenShift audit log and
+       alert on the organizational defined audit events.
```

Comment:
The control language does not mention active alerting, only enforcement of circumstances/usage conditions:

AC-2(11) | ACCOUNT MANAGEMENT \| USAGE CONDITIONSThe information system enforces [Assignment: organization-defined circumstances and/or usage conditions] for [Assignment: organization-defined information system accounts]. Supplemental Guidance: Organizations can describe the specific conditions or circumstances under which information system accounts can be used, for example, by restricting usage to certain days of the week, time of day, or specific durations of time.
-- | --

Is the language around OpenShift alerting needed?


``

Reply:
@shawndwells We can take it out.  I used the supplemental guidance for Rev 5 but Rev 4 doesn't have that verbiage.  If you're using Rev 4, then go with what you have.

Reply:
@mradecker ok, took it out. 

Updating everything to rev5 (once finalized) will be suuuuuper fun.